crates/interpreter/src/interpreter.rs

```diff
@@ -68,6 +69,29 @@ impl<EXT: Default, MG: MemoryGetter> Interpreter<EthInterpreter<EXT, MG>> {
             extend: EXT::default(),
         }
     }
+    pub fn with_bytecode(&mut self, bytecode: Bytecode) -> &Self {
```

Comment:
Builder pattern for `with_` function looks like
```suggestion

    pub fn with_bytecode(self, bytecode: Bytecode) -> Self {
```

Reply:
Done